progress/progress.go

```diff
@@ -188,4 +188,5 @@ func (b *Bar) overwrite(prefix string) {
 		clear = "\033[K"
 	)
 	fmt.Fprintf(b.out, "\r%v%v%v%v/%v %v\n", up, clear, prefix, b.i, b.n, b.t)
+	// fmt.Fprintf(b.out, "\r%v%v/%v %v\033[K", prefix, b.i, b.n, b.t)
```

Comment:
Why is this here?

Reply:
This should be removed, as it was during debugging/testing.

Here's a little background about the implementations:  the initial idea was to just use `\r` to write the line, but this doesn't clear the line, so any status updates whose print statement was shorter than a previous statement resulted in dangling characters at the end from the previous print.  Adding the control character `\033[K` resolved this by explicitly clearing the line. 

This was not sufficient, however. Lines also do not include a trailing linebreak, so any status or informative text printed during execution was overwritten, the began as being appended to the status line rather than on their own line.  By using a two-line solution the error can always start after the status indication line.